en/components/grids_templates/advanced_filtering.md

```diff
@@ -110,6 +110,70 @@ In case you don't want to show the @@igComponent toolbar, you could use the [`op
 >[!NOTE]
 >You can enable both the [`quickFilter`]({environment:angularApiUrl}/enums/filtermode.html#quickfilter)/[`excelStyleFilter`]({environment:angularApiUrl}/enums/filtermode.html#excelstylefilter) and the advanced filtering user interfaces in the @@igComponent. Both filtering user interfaces will work independently of one another. The final filtered result in the @@igComponent is the intersection between the results of the two filters.
 
+
+### External Advanced filtering
+
+As you see the demo above the appearance of the Advanced filtering dialog is inside the Infragistics grid. When the setup in the dialog is ready, the apply or close actions would hide that dialog. There is a way to make that dialog stay always visible - it can  be used outside of the grid as a standalone component. At the demo below, the advanced filtering dialog is declared separately of the grid.
```

Comment:
- The advanced filtering dialog is not exactly inside the grid. You could say 'the Advanced filtering dialog is hosted in an overlay on top of the @@igComponent'.
- There is double space 'can  be'.
- Modify 'be used outside of the grid as a standalone component' to 'be used as a standalone component'.
- '**At** the demo below' should be '**In** the demo below'.
- 'separately of the grid' should be 'separately of the @@igComponent'.